README.md

```diff
@@ -1,2 +1,88 @@
-# ansible-role-zammad
-Ansible role to install Zammad (https://zammad.org)
+ansible-role-zammad
+===================
+
+This role installs and/or configures Zammad, known from https://zamma.org .
+
+
+Requirements
+------------
+
+To run zammad, you have to meet the requirements from
+https://docs.zammad.org/en/latest/prerequisites-software.html .
+
+Please note that is highly recommended *not* to run the database server
+on the same machine as zammad itself.
+
+
+Role Variables
+--------------
+
+    # Install prerequisites via apt (disable and do it manually to use another package manager)
+    zammad_install_prerequisites: true
+
+    # If set to true, the setup process is triggered.
+    zammad_install: true
+
+    # If set to true, the configuration process is done.
```

Comment:
Is this sentence correct? When i read it, it sounds like a description for an output => 'i have configured everything, now i am done!'

Reply:
maybe like with the variable before `triggered` instead of `done` would do the trick...